spec/defines/init_spec.rb

```diff
@@ -42,6 +42,26 @@
     it { is_expected.to create_group('group2') }
   end
 
+  context 'managing gid only' do
+    let (:params) do
+      {
+        :state         => 'present',
+        :comment       => 'Rob Nelson',
+        :groups        => ['group1', 'group2'],
+        :password      => 'encryptedstring',
+        :manage_groups => 'gid',
+      }
+  end
+
+    it { is_expected.to create_user('rnelson0').with({
+      :comment          => 'Rob Nelson',
+      :groups           => ['group1', 'group2'],
+    }) }
+    it { is_expected.to create_group('rnelson0') }
+    it { is_expected.not_to create_group('group1') }
+    it { is_expected.not_to create_group('group2') }
+  end
+
```

Comment:
Add a test for `disabled|false` and `garbage` values, testing for no managed groups or an error code.